k2/torch/csrc/fsa_class_test.cu

```diff
@@ -22,49 +22,48 @@
 
 #include "gtest/gtest.h"
 #include "k2/csrc/fsa_algo.h"
-#include "k2/python/csrc/torch/torch_util.h"
-#include "k2/python/csrc/torch/v2/ragged_any.h"
-#include "k2/python/csrc/torch/v2/ragged_arc.h"
-#include "pybind11/embed.h"
+#include "k2/torch/csrc/fsa_class.h"
+#include "k2/torch/csrc/ragged_any.h"
+#include "k2/torch/csrc/torch_utils.h"
 
 namespace k2 {
 
-TEST(RaggedArcTest, FromUnaryFunctionTensor) {
+TEST(FsaClassTest, FromUnaryFunctionTensor) {
   for (const ContextPtr &c : {GetCpuContext(), GetCudaContext()}) {
     std::string s = R"(0 1 2 10
         0 1 1 20
         1 2 -1 30
         2)";
 
     auto device = GetDevice(c);
-    RaggedArc src = RaggedArc(s).To(device);
+    FsaClass src = FsaClass(s).To(device);
     src.SetRequiresGrad(true);
 
     src.SetAttr(
         "float_attr",
-        py::cast(torch::tensor(
+        torch::IValue(torch::tensor(
             {0.1, 0.2, 0.3},
             torch::dtype(torch::kFloat32).device(device).requires_grad(true))));
 
     src.SetAttr("int_attr",
-                py::cast(torch::tensor(
+                torch::IValue(torch::tensor(
                     {1, 2, 3}, torch::dtype(torch::kInt32).device(device))));
 
-    src.SetAttr("ragged_attr",
-                py::cast(RaggedAny("[[1 2 3] [5 6] []]",
-                                   py::cast(torch::kInt32), device)));
+    RaggedAny ragged_attr("[[1 2 3] [5 6] []]", torch::kInt32, device);
 
-    src.SetAttr("attr1", py::str("src"));
-    src.SetAttr("attr2", py::str("fsa"));
+    src.SetAttr("ragged_attr", ToIValue(ragged_attr));
+
+    src.SetAttr("attr1", torch::IValue("src"));
+    src.SetAttr("attr2", torch::IValue("fsa"));
```

Comment:
Only one test case for `std::string` is enough. I would suggest changing the second
case to test other types, e.g., `int`.